src/openms/source/ANALYSIS/OPENSWATH/DIAScoring.cpp

```diff
@@ -201,17 +201,44 @@ namespace OpenMS
   }
 
   /// Precursor isotope scores
+  void DIAScoring::dia_ms1_isotope_scores(double precursor_mz, SpectrumPtrType spectrum,
```

Comment:
What happens if you do not have identification for the compound and miss the EmpiricalFormula? Is the score just not reported or what happens? Since in theory, OpenSWATH should be able to quantify nonetheless?

Reply:
That may be the problem why sum formula was reported as a string instead of EmpiricalFormula - but I am not sure.

Reply:
yep. that is why I left the old overload. In that case, an averagine model is used. There is a comment somewhere that this is not accurate for metabolites but better than nothing.

Reply:
there seems to be too much code duplication with function below